packages/mongo/mongo_driver.js

```diff
@@ -128,17 +128,12 @@ MongoConnection = function (url, options) {
   self._observeMultiplexers = {};
   self._onFailoverHook = new Hook;
 
-  var mongoOptions = _.extend({
-    db: { safe: true },
```

Comment:
Took some work to track this down but I believe the omission of `db: { safe: true }` here is, well, safe.

`safe` was originally added to Meteor in https://github.com/meteor/meteor/commit/eee0f744d497bf24db8578a95aa71d9753123b5c which hints that it was a supported option by `mongodb@1.1.11` (the version [in use at the same time](https://github.com/meteor/meteor/blob/eee0f744d497bf24db8578a95aa71d9753123b5c/admin/generate-dev-bundle.sh#L86)).

The Node Driver [`HISTORY.md`](https://github.com/mongodb/node-mongodb-native/blob/2.2/HISTORY.md) doesn't really mention that `safe` was ever removed.  Suffice to say, it hasn't existed in the MongoClient `Db` documentation in a long, long time (see [1.4 version of the docs](https://mongodb.github.io/node-mongodb-native/api-generated/db.html)).

It is not listed in the [current version of the docs](https://mongodb.github.io/node-mongodb-native/2.2/api/MongoClient.html) but it does seem to be [somewhat-expected in the code](https://github.com/mongodb/node-mongodb-native/blob/5d2b36ca5f1b3f7ccf51750387568367df0a40c1/lib/url_parser.js#L391-L395) for URI connection strings, but [not documented](https://docs.mongodb.com/v3.2/reference/connection-string/) and the option should never make it to a server right now as it's being parsed in all three [Mongo Server](https://github.com/mongodb/node-mongodb-native/blob/791728ef40cc5b6a1a433dd79f1f26820a6259c2/lib/server.js#L46-L54), [Mongos](https://github.com/mongodb/node-mongodb-native/blob/791728ef40cc5b6a1a433dd79f1f26820a6259c2/lib/mongos.js#L49-L56) and [ReplSet](https://github.com/mongodb/node-mongodb-native/blob/791728ef40cc5b6a1a433dd79f1f26820a6259c2/lib/replset.js#L42-L49).

Reply:
I think this should be replaced with `w` and `j`, no? Allowing one to set this could address: https://github.com/meteor/meteor-feature-requests/issues/17